src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. So, you need to prepare data in App component and pass it to the child components.